oneanddone/users/tests/test_views.py

```diff
@@ -103,3 +103,12 @@ def test_get_object_non_existent_username(self):
 
         with self.assertRaises(Http404):
             self.view.get_object()
+
+    def test_get_object_existing_userid(self):
```

Comment:
Please move this test above `test_get_object_non_existent_username` (to keep them in alphabetical order).